.github/workflows/ci.yaml

```diff
@@ -58,17 +59,13 @@ jobs:
     name: Integration and e2e tests
     runs-on: ubuntu-latest
     timeout-minutes: 10
+    needs: [ "build", "chart-lint" ]
```

Comment:
The build is not needed for the chart to be linted and tested.

This way the pipeline can give you two errors (chart and build) and you can fix issues faster than if you put them sequentially. The pipeline is also faster this way because it parallelizes.

Reply:
I meant just to change the order of the section in the file

Reply:
Now I get it. I would say it is not a nit if this make the pipeline more readable.

Solved in [#ade2b8f](https://github.com/newrelic/newrelic-k8s-metrics-adapter/pull/40/commits/ade2b8f351eed5b932573105ecebe10d2240de47)